riff-raff/test/MigrateCDConfigs.scala

```diff
@@ -0,0 +1,7 @@
+import persistence.{ContinuousDeploymentConfigRepository, Persistence}
+
+object MigrateCDConfigs {
+  def main(args: Array[String]) {
+    Persistence.store.getContinuousDeploymentList().foreach(ContinuousDeploymentConfigRepository.setContinuousDeployment)
+  }
+}
```

Comment:
Historically I've done this in the main app by checking a migration key using this collection in mongo: https://github.com/guardian/riff-raff/blob/master/riff-raff/app/persistence/datastore.scala#L64-L66

Then run the migration as a one off if the key is missing.

This would also work.